features/support/env.rb

```diff
@@ -1,6 +1,9 @@
 require 'aruba/cucumber'
 
 Before do
+  # Force ids to be printed unquoted for consistency
+  set_env('SHELL', '/usr/bin/bash')
```

Comment:
It's related to the shell detection elsewhere, but I guess you have to be familiar with that for this comment to be ok